templates/zerver/help/enable-high-contrast-mode.md

```diff
@@ -0,0 +1,12 @@
+# Enable high contrast mode
+
+By default, the date/time stamps are intentionally light in order to avoid
+having too much emphasis. However, you can choose to increase the contrast of
+the user interface by using the following procedure:
+
+{!go-to-the.md!} [Display settings](/#settings/display-settings)
+{!settings.md!}
+
+2. Select the option labeled **Enable high contrast mode**.
+
+TODO: insert screenshots of what low-contrast and high-contrast mode look like
```

Comment:
Since I'd like to merge this PR, I think we should make a couple changes here:
(1) remove the TODO, and track that work elsewhere.
(2) Explain the feature more clearly.  "accessibility" should be in the title and the description of the purpose (maybe with some preamble about how Zulip is designed with accessibility in mind and aims to be a leader in accessibility practices; we can link to the RTD page if we like?).